aws-predictions-tensorflow/src/main/java/com/amplifyframework/predictions/tensorflow/service/TensorFlowPredictionsService.java

```diff
@@ -35,7 +36,7 @@
  * pre-trained models to make predictions offline.
  */
 public final class TensorFlowPredictionsService {
-    private static final Logger LOG = Amplify.Logging.forNamespace("amplify:aws-predictions-tensorflow");
+    private static final Logger LOG = Amplify.Logging.forCategory(CategoryType.PREDICTIONS);
```

Comment:
This is an interesting one as there will be routinely two plugins -- `aws-predictions` and `aws-predictions-tensorflow` running concurrently.

Reply:
The original format here was both like `ampilfy:plugin-name` and `amplify-category-name`. 

Unfortunately, `forPlugin(Plugin<?> plugin)` turned out not to be a realistic option. It would mean we'd have to have the plugin instance available in order to call the logger. 

I don't love putting everything under the categroy slug, but at least its consistent.

I'll redo this PR and just make it about the default log level, not about the slug stuff.